apps/genetics/src/components/NavBar/NavBar.jsx

```diff
@@ -94,58 +124,63 @@ const NavBar = ({
       color="primary"
       elevation={0}
     >
-      <Toolbar variant="dense">
-        {homepage ? null : (
-          <Button component={ReactRouterLink} to="/" color="inherit">
-            <OpenTargetsTitle name={name} />
-          </Button>
-        )}
-        <div className={classes.flex} />
-        {search ? search : null}
-
-        {docs ? (
-          <MenuExternalLink classes={classes} href={docs}>
-            Docs
-          </MenuExternalLink>
-        ) : null}
+      {/* push the content down so it isn't hidden behind the logo bar */}
+      {config.showTopBar &&
+          <div id="placeholder-div" style={{ height: '50px', width: '100%' }} />}
+      <Toolbar variant="dense" className={classes.spaceBetween}>
```

Comment:
Do these lines (apart from the one above that inserts a div) have to change? I don't think they had to before, and it looks like a lot more of a hassle to maintain when EBI releases new versions, especially if they shouldn't change for the platform version.

Reply:
See [discussion under the other occurrence of this code](https://github.com/thehyve/ot-ui-apps/pull/2#discussion_r1171458452).